src/sh/keptn/Keptn.groovy

```diff
@@ -222,26 +247,23 @@ def keptnInit(Map args) {
                 | "type": "sh.keptn.event.monitoring.configure"
                 |}
             """.stripMargin()
-            def configureMonitoringResponse = httpRequest contentType: 'APPLICATION_JSON', 
-                customHeaders: [[maskValue: true, name: 'x-token', value: "${keptn_api_token}"]], 
-                httpMode: 'POST', 
-                requestBody: configureMonitoringBody, 
-                responseHandle: 'STRING', 
-                url: "${keptn_endpoint}/v1/event", 
+        def configureMonitoringResponse = httpRequest contentType: 'APPLICATION_JSON',
+                customHeaders: [[maskValue: true, name: 'x-token', value: "${keptn_api_token}"]],
+                httpMode: 'POST',
+                requestBody: configureMonitoringBody,
+                responseHandle: 'STRING',
+                url: "${keptn_endpoint}/v1/event",
                 validResponseCodes: "100:404",
                 ignoreSslErrors: true
 
-            if (configureMonitoringResponse.status == 200) {
-                echo "Successfully configured monitoring for: ${monitoring}"
-                echo "body: ${configureMonitoringBody}"
-            } else {
-                echo "Couldnt configure monitoring for ${monitoring}: " + configureMonitoringResponse.content
-                echo "body: ${configureMonitoringBody}"
-            }    
+        if (configureMonitoringResponse.status == 200) {
+            echo "Successfully configured monitoring for: ${monitoring}"
+            echo "body: ${configureMonitoringBody}"
+        } else {
+            echo "Couldnt configure monitoring for ${monitoring}: " + configureMonitoringResponse.content
+            echo "body: ${configureMonitoringBody}"
```

Comment:
I know you this was already in the existing code, but I will create an issue to mimic the CLI behaviour here, which means that we need to wait for a `sh.keptn.event.monitoring-configure.finished` cloudevent.

Reply:
Tracked by #94